### ISSUE TITLE
ValueError for normal forms

### ISSUE THREAD

**Issue by szilvajuhos** [ndaniel/fusioncatcher#177]
Cheers, this line needs an extra conversion (likely the other similar lines are also in need of fixing):

https://github.com/ndaniel/fusioncatcher/blob/5813f67bc075cb45fd2b2cdcfb5c836ac5e0b6ef/bin/fusioncatcher.py#L8228

and should be like

nucleotides_ggu = int(**float**(file(outdir('gene-gene__nuc.txt'),'r').readline().strip()))

to avoid type conversion errors. Now we have 
```
>>> int(file('gene-gene__nuc.txt','r').readline().strip())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '2.43432e+09'
```
with float() it would be:
```
>>> int(float(file('gene-gene__nuc.txt','r').readline().strip()))
2434320000
```

**Comment by ndaniel** [comment]
Good point!

**Comment by ndaniel** [comment]
This will will/should be fixed and v1.30.
